### PR TITLE
📖 Fix callouts breaking grow

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

File: spec/amp-cors-requests.md [hunk 1]
```diff
@@ -145,12 +145,7 @@ For example, endpoints should allow requests from:
 - Google AMP Cache subdomain: `https://<publisher's domain>.cdn.ampproject.org` <br>(for example, `https://nytimes-com.cdn.ampproject.org`)
 - Cloudflare AMP Cache: `https://<publisher's domain>.amp.cloudflare.com`
 
-{% call callout('Read on', type='read') %}
-For information on AMP Cache URL formats, see these resources:
-
-- [Google AMP Cache Overview](https://developers.google.com/amp/cache/overview)
-- [Cloudflare AMP Cache](https://amp.cloudflare.com/)
-  {% endcall %}
+Read on: For information on AMP Cache URL formats, see these resources: [Google AMP Cache Overview](https://developers.google.com/amp/cache/overview) and [Cloudflare AMP Cache](https://amp.cloudflare.com/).
```
Comment (rsimha) [inline spec/amp-cors-requests.md]: I think the words `Read on: ` can be dropped here.

File: spec/amp-cors-requests.md [hunk 2]
```diff
@@ -181,9 +176,7 @@ Although the W3 CORS spec allows the value of <code>\*</code> to be returned in
 
 ### Processing state changing requests
 
-{% call callout('Important', type='caution') %}
-Perform these validation checks _before_ you process the request. This validation helps to provide protection against CSRF attacks, and avoids processing untrusted sources requests.
-{% endcall %}
+Important: Perform these validation checks _before_ you process the request. This validation helps to provide protection against CSRF attacks, and avoids processing untrusted sources requests.
```
Comment (rsimha) [inline spec/amp-cors-requests.md]: Same here. I think `Important: ` can be dropped.

File: spec/amp-cors-requests.md [hunk 3]
```diff
@@ -402,9 +395,7 @@ As an example, if you wanted to load /some/font.ttf in `https://example.com/amp.
   </noscript>
 </amp-img>
 
-[tip type="note]
-If your font file is okay to be accessible from any origin, you can respond with a wild card `Access-Control-Allow-Origin`, AMP cache will also echo that value meaning it will be responding with `Access-Control-Allow-Origin: *`. If you already have this setting, there is no need in changing anything.
-[/tip]
+Note: If your font file is okay to be accessible from any origin, you can respond with a wild card `Access-Control-Allow-Origin`, AMP cache will also echo that value meaning it will be responding with `Access-Control-Allow-Origin: *`. If you already have this setting, there is no need in changing anything.
```
Comment (rsimha) [inline spec/amp-cors-requests.md]: And same here. Drop the words `Note: `.

File: spec/amp-cors-requests.md (hunk 1 above)
Comment (sebastianbenz) [reply spec/amp-cors-requests.md]: Nope. That's an amp.dev shortcut which will result in a visual callout. The good thing about this is that it's still readable in markdown.

**Comment by sebastianbenz** [comment]
@CrystalOnScript the callouts are for the visual callouts on amp.dev. This is what `Read on: ...` will look like:

Screen Shot 2019-11-22 at 20 47 58

**Comment by mrjoro** [comment]
Is this ready to merge (after resolving conflicts)?

**Pull request closed without merging**